client/routes/TodoPage.tsx

```diff
@@ -0,0 +1,15 @@
+// src/routes/TodosPage.tsx
+import { TodoList } from "@client/components/TodoList";
+import { TodoForm } from "@client/components/todoForm"; // Getting error here, WHY???
+
+const TodosPage = () => {
```

Comment:
files in the routes directory map to actual frontend routes and need to have `createFileRoute` like about and index do